includes/ApplHandlingByID.php

```diff
@@ -299,6 +299,163 @@ function accept ( $id, $type, $user )
+		$newStatus = $status[0]; // to create the new status.
+		$flag = 0;       // a flag to check wether the status has been updated or not.
+		for ( $i = 1 ; $i < count ( $status ) ; $i++ )
+		{
+			$Stat = explode ( ',', $status[$i] );
+			if ( strcmp ( $user, $Stat[0] ) == 0 )
+			{
+				$flag = $i;
+				if ( $Stat[1] == 'P' )
+				{
+					$Stat[1] = 'F';
+					$status[$i] = $Stat[0].",".$Stat[1];
+				}
+			}
+			$newStatus = $newStatus.";".$status[$i];
+		}
+		if ( $flag != 0 )
+		{
+			$QRY = "UPDATE $type SET $stat = ? WHERE $AppId = ?";
+			$STMT = $sqlconn->prepare ( $QRY );
+			$STMT->bind_param ( 'ss', $newStatus, $id );
+			if ( ! $STMT->execute() )
+			{
+				echo " Unable to perform the task, internal server error.<br>";
+				die();
+			}
+			$next = explode ( ',', $status[ $flag + 1 ] );
+			$NEXT = str_replace('.','$', $next[0] );
+			$STMT  = $sqlconn->prepare ( "INSERT INTO $NEXT VALUES ( ?,?,? )" );
+			$STMT->bind_param ( 'sss', $id, $type, $row[1] );
+			if ( ! $STMT->execute() )
+			{
+				echo " Unable to perform the task, internal server error.<br>";
+				die();
+			}
+			$STMT->close();
+
+		}
+		else
+		{
+			return false;
+		}
+	}
+	$stmt->close();
+	
+	return true;
+}
+
+
+
+function forwardAcceptor ( $id, $type, $user, $next )
+{
+	require "../../LocalSettings.php";
+	require "../Globals.php";
+
+	$sqlconn = new mysqli ( $eorgDBserver, $eorgDBuser, $eorgDBpasswd, $eorgDBname );
+
+	if ( $sqlconn->connect_errno )
+	{
+		echo "Internal Server Error 3, Sorry for inconvenience.";
+		die();
+	}
+
+	$qry = "SELECT $stat, $AppDate FROM $type WHERE $AppId = ?";
+	$stmt = $sqlconn->prepare ( $qry );
+
+	$stmt->bind_param ( 's', $id );
+	if ( ! $stmt->execute() )
+	{
+		echo "Sorry, there is a problem with database.";
+		die();
+	}
+	else
+	{
+		$result = $stmt->get_result();
+		$row = mysqli_fetch_row ( $result );
+		$status = explode ( ';', $row[0] );
+		$newStatus = $status[0]; // to create the new status.
+		$flag = 0;       // a flag to check wether the status has been updated or not.
+		for ( $i = 1 ; $i < count ( $status ) ; $i++ )
+		{
+			$Stat = explode ( ',', $status[$i] );
+			if ( strcmp ( $user, $Stat[0] ) == 0 )
+			{
+				$flag = $i;
+				if ( $Stat[1] == 'P' )
+				{
+					$Stat[1] = 'F';
+					$status[$i] = $Stat[0].",".$Stat[1];
+				}
+			}
+			$newStatus = $newStatus.";".$status[$i];
+		}
+		
+		// set next user and status
+		$newStatus = $newStatus.";".$next.",P";
+		
+		if ( $flag != 0 )
+		{
+			$QRY = "UPDATE $type SET $stat = ? WHERE $AppId = ?";
+			$STMT = $sqlconn->prepare ( $QRY );
+			$STMT->bind_param ( 'ss', $newStatus, $id );
+			if ( ! $STMT->execute() )
+			{
+				echo " Unable to perform the task, internal server error.<br>";
+				die();
+			}
+			
+			$STMT  = $sqlconn->prepare ( "INSERT INTO $next VALUES ( ?,?,? )" );
+			$STMT->bind_param ( 'sss', $id, $type, $row[1] );
+			if ( ! $STMT->execute() )
+			{
+				echo " Unable to perform the task, internal server error.<br>";
+				die();
```

Comment:
__[phpmd]__
* The function forwardAcceptor() contains an exit expression.

<!-- Comment by [SideCI](https://sideci.com) -->